packages/rulesets/generated/spectral/az-dataplane.js

```diff
@@ -985,6 +991,8 @@ const ruleset$1 = {
             description: "The value of the 'description' property must be descriptive. It cannot be spaces or empty description.",
             message: "'{{property}}' parameter lacks 'description' property. Consider adding a 'description' element. Accurate description is essential for maintaining reference documentation.",
             severity: "error",
+            disableForTypeSpec: true,
+            disableForTypeSpecReason: "TODO",
```

Comment:
```suggestion
            disableForTypeSpecReason: "Covered by TSP's '@azure-tools/typespec-azure-core/documentation-required' rule.",
```

I believe this is the same rule

Reply:
Actually we don't have this validation in place on the TSP side:
https://github.com/Azure/typespec-azure-pr/issues/3882

I don't think we should disable this rule yet.

Reply:
Oh I thought we had that rule but yeah apparently not. So yeah lets' not remove yet.